src/RenderWebGL.js

```diff
@@ -223,6 +223,8 @@ class RenderWebGL extends EventEmitter {
         const pixelRatio = window.devicePixelRatio || 1;
         this._gl.canvas.width = pixelsWide * pixelRatio;
         this._gl.canvas.height = pixelsTall * pixelRatio;
+        // Resizing the canvas can cause it to be cleared, so redraw it.
+        this.draw();
```

Comment:
I like this idea! It seems unnecessary to resize & redraw every frame when the color picker is active, though. Are you aware of a reason we need to do that? If not, what do you think about something like this?
```suggestion
        const newWidth = pixelsWide * pixelRatio;
        const newHeight = pixelsTall * pixelRatio;
        if (this._gl.canvas.width != newWidth || this._gl.canvas.height != newHeight) {
            this._gl.canvas.width = pixelsWide * pixelRatio;
            this._gl.canvas.height = pixelsTall * pixelRatio;
            // Resizing the canvas can cause it to be cleared, so redraw it.
            this.draw();
        }
```

Reply:
I was hesitant to include that in this PR in order to keep changes minimal considering it's a patch for a critical bug, but that's a good idea.

Reply:
Ah, good point. I think this is OK but I admire your restraint! :D